yarn.lock

```diff
@@ -5124,7 +5124,7 @@ __metadata:
 
 "@lezer/highlight@npm:^1.0.0, @lezer/highlight@npm:^1.1.3, @lezer/highlight@npm:^1.2.0":
   version: 1.2.0
-  resolution: "@lezer/highlight@npm:1.2.0"
+  resolution: "@lezer/highlight@npm:1.2.1"
   dependencies:
     "@lezer/common": ^1.0.0
   checksum: 5b9dfe741f95db13f6124cb9556a43011cb8041ecf490be98d44a86b04d926a66e912bcd3a766f6a3d79e064410f1a2f60ab240b50b645a12c56987bf4870086
```

Comment:
We also need to update the checksums.

Reply:
Is it done by using `yarn install --update-checksums`? I currently don't have `yarn` locally, so I need to install it and perform the checksum update right?

Reply:
JupyterLab comes with a bundled version of `yarn` under `jlpm` alias. The syntax you mention is for `yarn` v1 so it would not work here (`jlpm` points to yarn v3). Instead you can run:

```
YARN_CHECKSUM_BEHAVIOR=update jlpm
```

from an environment where jupyterlab is installed (and commit the changes).

Reply:
I was struggling to run the command, but later found that `set YARN_CHECKSUM_BEHAVIOR=update` then `jlpm` works on Windows. I have pushed a new commit. Should I also squash the 2 commits into 1?

Reply:
No, it's fine - we use squash merging.